src/api/controllers/pluginController.ts

```diff
@@ -16,42 +23,15 @@ import * as crypto from 'crypto';
  */
 const getPlugins = async (req: Request, res: Response) => {
   const { pageSize = 30, pageNum = 1, searchQuery = '' } = req.query;
-  const limit = parseInt(pageSize as string, 30);
-  const offset = (parseInt(pageNum as string, 30) - 1) * limit;
-  const whereClause = searchQuery
-    ? {
-        [Op.or]: [
-          { name: { [Op.like]: `%${searchQuery}%` } },
-          { description: { [Op.like]: `%${searchQuery}%` } },
-        ],
-      }
-    : {};
+  const limit = parseInt(pageSize as string) || 30;
+  const offset = ((parseInt(pageNum as string) || 30) - 1) * limit;
 
   try {
-    const foundPlugins =await Plugin.findAll({
-      where: whereClause,
-      offset,
+    const plugins = await getPluginsService({
+      searchQuery: searchQuery as string,
       limit,
-      raw: true
-    })
-
-    const plugins = await Promise.all(foundPlugins.map(async (plugin) => {
```

Comment:
is this intended? Looks like you removed the whole logic for creating a presignedurl. Is this complete ?

Reply:
Yes it is intended. Currently our implementation will have the buckets be public so no need to create presigned URLs, if we decide that we dont want public URLs we would have to change a few things